public/media_engagement/imaqal_media_engagement.js

```diff
@@ -36,6 +36,9 @@ firebase.auth().onAuthStateChanged(function (user) {
     }
 });
 
+const TIMEFRAME = 7
+var view = "1day"
```

Comment:
I think we could probably improve on the variable name 'view':

- What is it a selection of?
- What are the possible values?